pydicom/tests/test_uid.py

```diff
@@ -1,93 +1,22 @@
-# test_uid.py
 """Test suite for uid.py"""
-# Copyright (c) 2008-2012 Darcy Mason
-# This file is part of pydicom, released under a modified MIT license.
-#    See the file LICENSE included with this distribution, also
-#    available at https://github.com/pydicom/pydicom
-
```

Comment:
I've open a new issue (#447) to consolidate this discussion in a separate process to find it easier later.  Basically, I think it is time to review this and find what current practice indicates.